regex-assembly/942370.ra

```diff
@@ -36,7 +36,7 @@
   ##!<
   [^\w\s?]+\s*?[^\w\s]+\s*?{{quotes}}
   [^\w\s]+\s*?[\W\d][^#]*#
-  [^\w\s]+\s*?[\W\d][^-]*--
+  [^\w\s]+\s*?[\W\d][^-]*(-){2}
```

Comment:
That's still the same expression. This can be bypassed by adding a single `-`. See https://regex101.com/r/t8iDOa/1.